command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
+rucio did content list
+rucio did content history
+rucio did content close
+rucio did content open
```

Comment:
I wonder about this. Whether a DID is open is part of the DID itself (meaning a `did update` command), not its content. But then what would `did content update` do? Needs more thought.

Reply:
I think because open/close are key operations you'd do with content in general (you can open and close boxes, doors to buildings, datasets), it's more intuitive than mapping several operations to update.

Reply:
You make a very nice visual example and I took it into consideration, but I don’t think that it adapts well in practice.

1. Closing a DID can only be done once and users rarely do it. Workflow management systems use the API. Regular users don’t have the permission to open a DID.
2. As I wrote before, when opening or closing a DID, you are manipulating the `dids` table. When you are attaching or detaching DIDs, you are manipulating the `contents` table. It is more consistent for the interface to follow the models.
3. It is actually more intuitive to map all fields that appear in `did show` to `did update`. I see `is_open: False`, and I can modify it with `--close` (or whatever else). Otherwise, you can’t deduce from any `did content` command whether it is open or not, unless you try to attach a DID and see an error.